src/services/edusharing/services/EduSharingConnectorV7.js

```diff
@@ -221,11 +227,56 @@ class EduSharingConnector {
 		}
 	}
 
+	async getPlayerForNode(nodeUuid) {
+		const url = `${ES_ENDPOINTS.RENDERER}${nodeUuid}`;
+		const options = {
+			method: 'GET',
+			url,
+			headers: {
+				Accept: 'application/json',
+			},
+		};
+		const user = Configuration.get('ES_USER');
+
+		const response = await this.eduSharingRequest(options, user);
+		const parsed = JSON.parse(response);
+		if (parsed && typeof parsed.detailsSnippet === 'string') {
+			return this.getH5Piframe(parsed.detailsSnippet);
+		}
+		throw new Unavailable(`Unexpected response from Edu-Sharing renderer.`);
+	}
+
 	validateUuid(uuid) {
 		const uuidV5Regex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
 		const uuidV4Regex = /^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i;
 		return uuidV4Regex.test(uuid) === true || uuidV5Regex.test(uuid) === true;
 	}
+
+	getH5Piframe(html) {
+		const cleanTags = sanitizeHtml(html, {
+			allowedTags: ['iframe', 'script'],
+			allowVulnerableTags: true,
+			allowedAttributes: {
+				iframe: ['src'],
+				script: ['src'],
+			},
+		});
+
+		const iframeSrc = /<iframe src="(.*?)"><\/iframe>/g.exec(cleanTags);
+		const scriptSrc = /<script src="(.*?)"><\/script>/g.exec(cleanTags);
+
+		if (
+			iframeSrc === null ||
```

Comment:
iframeSrc === null is not needed anymore, is done by the Array.isArray check

Reply:
Done